charts/nri-bundle/Chart.yaml

```diff
@@ -58,7 +58,7 @@ dependencies:
   - name: newrelic-logging
     repository: https://newrelic.github.io/helm-charts
     condition: logging.enabled,newrelic-logging.enabled
-    version: 1.23.0
+    version: 1.23.1
```

Comment:
I think this gets updated through an automation, refer to this PR https://github.com/newrelic/helm-charts/pull/1512. You should cancel this.